package.json

```diff
@@ -116,8 +119,7 @@
     "prettyjson": "1.2.1",
     "shelljs": "0.8.5",
     "sinon": "12.0.1",
-    "webpack": "5.24.2",
```

Comment:
So we don't need webpack anymore? Which means no minification and no bundling anymore?

Reply:
exactly, the modules in src/ are transpiled into the modules in lib/, no bundling just native nodejs module loading.